### PR TITLE
Update Dataflow containers

### PR DESCRIPTION
**Pull request by ahmedabu98**
Updating to include the dependency updates in #28880

### CODE REVIEW

File: website/www/site/content/en/documentation/runtime/environments.md [hunk 1]
```diff
@@ -115,14 +115,13 @@ This method requires building image artifacts from Beam source. For additional i
...
   ./gradlew :sdks:go:container:docker
-  ./gradlew :sdks:python:container:py36:docker
   ./gradlew :sdks:python:container:py38:docker
   ./gradlew :sdks:python:container:py39:docker
   ./gradlew :sdks:python:container:py310:docker
   ./gradlew :sdks:python:container:py311:docker
 
   # Shortcut for building all Python SDKs
-  ./gradlew :sdks:python:container buildAll
+  ./gradlew :sdks:python:container:buildAll
```
Comment (johnjcasey) [inline website/www/site/content/en/documentation/runtime/environments.md:124]: is this change intentional?

Comment (ahmedabu98) [reply website/www/site/content/en/documentation/runtime/environments.md:124]: yes, `./gradlew :sdks:python:container buildAll` is not a real command. I suspect the space was by mistake

**Review by Abacn — APPROVED** [review]
Thanks for fixing the documentation
